eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -28,7 +28,7 @@ parameters:
   default: 'sdk/*/*/*cov_xml.xml'
 - name: LiveTestTimeoutInMinutes
   type: number
-  default: 60
+  default: 90
```

Comment:
Is this intended to be permanent? We try and reduce test times as opposed to increasing the timeout unless there is a real need to run tests for a longer period.